src/Generators/DataTablesMakeCommand.php

```diff
@@ -72,14 +79,83 @@ protected function getModel()
     {
         $name           = $this->getNameInput();
         $rootNamespace  = $this->laravel->getNamespace();
-        $modelNamespace = $this->laravel['config']->get('datatables-buttons.namespace.model');
+        $model = $this->option('model') || $this->option('model-namespace');
+        $modelNamespace = $this->option('model-namespace') ? $this->option('model-namespace') : $this->laravel['config']->get('datatables-buttons.namespace.model');
 
-        return $this->option('model')
-            ? $rootNamespace . "\\" . ($modelNamespace ? $modelNamespace . "\\" : "") . str_singular($name)
+        return $model
+            ? $rootNamespace . "\\" . ($modelNamespace ? $modelNamespace . "\\" : "") .  str_singular($name)
             : $rootNamespace . "\\User";
     }
 
     /**
+     * Replace columns.
+     *
+     * @param string $stub
+     * @return $this
+     */
+    protected function replaceColumns(&$stub)
+    {
+        $stub = str_replace(
+            'DummyColumns', $this->getColumns(), $stub
+        );
+
+        return $this;
+    }
+
+    /**
+     * Get the columns to be used.
+     *
+     * @return string
+     */
+    protected function getColumns()
+    {
+        if($this->option('columns') != ''){
+            return $this->parseArray($this->option('columns'));
+        }else{
+            return $this->parseArray('id,add your columns,created_at,updated_at');
+        }
+    }
+
+    /**
+     * Parse array from definition
+     *
+     * @param  string  $definition
+     * @param  string  $delimiter
+     * @param  int     $indentation
+     * @return string
+     */
+    protected function parseArray($definition, $delimiter = ',', $indentation = 16)
+    {
```

Comment:
I think 12 is the proper indention here to have a perfectly aligned columns. `$indention = 12`